README.md

```diff
@@ -24,6 +24,37 @@ and renders declaratively using `lit-html`.
     * attribute: ``` html`<div class$="${color} special"></div>` ```
     * event handler: ``` html`<button on-click="${(e) => this._clickHandler(e)}"></button>` ```
 
+## Getting started
+
+ 1. The easiest way to get started using LitElement is to use one of these online tools:
+
+    * all [supported](#Supported-Browsers) browsers: [stackblitz](https://stackblitz.com/edit/hello-lit-element?file=index.js)
+
+    * browsers with [Javascript Modules](https://caniuse.com/#search=modules): [JSBin](http://jsbin.com/zezilad/edit?html,output),
+ [Glitch](https://glitch.com/edit/#!/hello-lit-element?path=index.html), or
+ [CodePen](https://codepen.io/sorvell/pen/BxZgPN).
+
+ 1. You can also copy [this HTML](https://gist.githubusercontent.com/sorvell/48f4b7be35c8748e8f6db5c66d36ee29/raw/352a3f6e2de37b60071faa3e75c3be96e49b0e92/index.html) into a local file and run it in any browser that supports [Javascript Modules]((https://caniuse.com/#search=modules)).
+
+ 1. For a local development, a development server is required. We recommend installing
+ the polymer-cli to and using its development server as follows.
+
+    * install the cli:
+
```

Comment:
Missing "import lit-element and extend it"

Reply:
Done